README.md

```diff
@@ -728,6 +729,30 @@ I will set the Heating setpoint of the node with id `4` located in the `office`
 
 `zwave/office/nodeID_4/thermostat_setpoint/heating`
 
+### Multicast
+
+You can send Multicast requests to _all values with a specific suffix_ of a _group_ of nodes in the network.
```

Comment:
I'm not really clear on this, maybe provide a use case from the outset such as adjusting all your thermostats/lights?

be clearer that this is just a shorthand, it's not an _actual multicast_, behind the scenes zj2m is doing the heavy lifting and iterating

Reply:
`zwave-js` does support multicast (if you use the same command and target the same endpoint of all nodes)

Reply:
I've raised https://github.com/zwave-js/node-zwave-js/issues/1150 to track that.

Reply:
For reference, what you can do already is this:
```ts
const cmd = new BinarySwitchCCSet(driver, {
	nodeId: [2,3,4,5],
	targetValue: true,
});
await driver.sendCommand(cmd);
```

Reply:
I think we should wait for:  zwave-js/node-zwave-js#1150 Before going on with this?

Reply:
> For reference, what you can do already is this:

@AlCalzone How could I get the correct command class constructor based on the command class id?

Reply:
Sorry, I didn't see this...
https://github.com/zwave-js/node-zwave-js/blob/daf0feb9bfe14ae5b9a24c0d0d53206d5bdd41e9/packages/zwave-js/src/lib/commandclass/CommandClass.ts#L1057-L1059

But I'm not sure if that is exported through the main entry point.